src/app/active/active.module.ts

```diff
@@ -0,0 +1,16 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { SharedModule } from '../shared/shared/shared.module';
+import { ActiveComponent } from './active.component';
+
+@NgModule({
+  declarations: [ ActiveComponent],
+  imports: [
+    CommonModule,
+    SharedModule
```

Comment:
`SharedModule` can be re-named to `sharedUsersModule`  which represents the module purpose